integration/conftest.py

```diff
@@ -35,6 +37,9 @@ def _get_cert_path(request):
                      '"--cert" command-line option.')
 
 def integration_conf(request):
+    project_id = request.config.getoption('--project')
+    if project_id:
+        return credentials.FakeCredential(), project_id
```

Comment:
Would it be possible to move this logic to `integration/test_db.py`? There we can have something like:

```
# integration/test_db.py

def db_integration_conf():
    host_override = os.environ.get('FIREBASE_DATABASE_EMULATOR_HOST')
    if host_override:
        return db._FakeCredential(), 'fake-project-id'
    return conftest.integration_conf()
```

Then make sure integration tests use an App instance created using this test configuration.

This way we don't need a separate global `--project` flag, and the changes are contained to the test_db file.

Reply:
Done.